### PR TITLE
Fix autonomous.

### PR DESCRIPTION
**Pull request by amackworth**
@col6y After running many, many tests, this _seems_ to work in the emulator now, but it's still really hard to tell. :sweat_smile:

### CODE REVIEW

File: QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java [hunk 1]
```diff
@@ -40,5 +42,6 @@ protected void autonomousMain() throws AutonomousModeOverException, InterruptedE
 		waitForTime(timeout);
 		
 		Rollers.running.set(false);
+		done.set(true);
```
Comment (celskeggs) [inline QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java]: What about if there's an error? Make sure that done gets set back to true. Or at least report something.

File: QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeBase.java [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 import ccre.instinct.InstinctModeModule;
 
 public abstract class AutonomousModeBase extends InstinctModeModule {
-	public FloatInputPoll driveSpeed;
-	public FloatInputPoll rotateSpeed;
-	public FloatInputPoll clampHeightPadding;
+	public static TuningContext context = ControlInterface.autoTuning;
+	public static FloatInputPoll driveSpeed = context.getFloat("auto-main-driveSpeed", 1.0f);
+	public static FloatInputPoll rotateSpeed = context.getFloat("auto-main-rotateSpeed", 1.0f);
+	public static FloatInputPoll clampHeightPadding = context.getFloat("auto-main-clampHeightPadding", 0.01f);
```
Comment (celskeggs) [inline QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeBase.java]: You can probably make most of these final. Maybe also private.

**Comment by celskeggs** [comment]
I feel like there should be an easier way to do the `done` thing, but I can't find it. Oh well.

**Comment by amackworth** [comment]
Yeah, same. Thought about it for a while, but just decided to go ahead with the naïve solution.

**Comment by celskeggs** [comment]
Looks good.

**Pull request merged**